temporalio/client.py

```diff
@@ -360,11 +380,17 @@ async def start_workflow(
         retry_policy: Optional[temporalio.common.RetryPolicy] = None,
         cron_schedule: str = "",
         memo: Optional[Mapping[str, Any]] = None,
-        search_attributes: Optional[temporalio.common.SearchAttributes] = None,
+        search_attributes: Optional[
+            Union[
+                temporalio.common.TypedSearchAttributes,
+                temporalio.common.SearchAttributes,
+            ]
+        ] = None,
         start_signal: Optional[str] = None,
         start_signal_args: Sequence[Any] = [],
         rpc_metadata: Mapping[str, str] = {},
         rpc_timeout: Optional[timedelta] = None,
+        stack_level: int = 2,
```

Comment:
Is this left over?

Reply:
No, it's intentional as a way to set warning stack. Arguably I shouldn't expose to users, but if they are a wrapper library/utility, they may want their warnings to be on outer stack too.